packages/opentelemetry-sdk-node/src/types.ts

```diff
@@ -38,5 +38,5 @@ export interface NodeSDKConfiguration {
   sampler: Sampler;
   spanProcessor: SpanProcessor;
   traceExporter: SpanExporter;
-  traceParams: TraceParams;
+  traceParams: SpanLimits;
```

Comment:
```suggestion
  spanLimits: SpanLimits;
```

Reply:
Thank you! Can't believe I missed these! I have updated the code